### ISSUE TITLE
Expressing conformance metadata

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#698]
The current proposal for expressing conformance requires one property from dcterms (conformsTo) and another developed specifically for epub (a11y:certifiedBy).

Ideally, we should have a related set of properties to express this information, but so far the group has not been able to find a single vocabulary that can express this information.

The group intends to continue discussions around this metadata and may propose a new method for expressing in a future draft.

**Comment by mattgarrish** [comment]
The accessibility group has decided to continue to use these properties, but they have been moved to the techniques document to allow changes in the future.

Trying to get these added to schema.org in the 3.1 timeline was not seen as realistic, and their definition not specifically accessibility related (i.e., outside the scope of this group to define).
